File: client-app/src/models/boardgame.ts
import { Profile } from "./profile"

export interface BoardGame {
    id: string
    title: string
    releaseDate: Date | null
    description: string
    category: string
    maxPlayers: number
    minPlayers: number
    boardGameOwners?: Profile[];
    isOwning: boolean;
  }
  

File: Boardgame/Domain/BoardGame.cs
namespace Domain
{
    public class BoardGame
    {
        public Guid Id { get; set; }
        public string Title { get; set; }
        public DateTime ReleaseDate { get; set; }
        public string Description { get; set; }
        public string Category { get; set; }
        public int MaxPlayers { get; set; }
        public int MinPlayers { get; set; }
        public ICollection<BoardGameOwner> BoardGameOwners { get; set; } = new List<BoardGameOwner>();
        
        
    }
}